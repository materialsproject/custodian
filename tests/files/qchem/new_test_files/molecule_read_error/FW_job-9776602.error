/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 2072 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 2891
DEBUG:launchpad:RUNNING FW with id: 2072
INFO:rocket.launcher:RUNNING fw_id: 2072 in directory: /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-04-03-30-43-540608/launcher_2018-07-07-05-35-08-084164
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-07-06 23:01:30.585748 in /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-04-03-30-43-540608/launcher_2018-07-07-05-35-08-084164.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid04602, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
/global/common/edison/software/qchem/5.1beta/bin/qchem: line 17: echo: write error: Input/output error
srun: error: nid04602: task 0: Aborted
srun: Terminating job step 9776602.0
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:root:Backing up run to error.1.tar.gz.
ERROR:custodian.custodian:
{ 'actions': None,
  'errors': [ 'unknown_error'],
  'handler': <custodian.qchem.handlers.QChemErrorHandler object at 0x2aaabd6619e8>}
ERROR:custodian.custodian:Unrecoverable error for handler: <custodian.qchem.handlers.QChemErrorHandler object at 0x2aaabd6619e8>. Raising RuntimeError
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-07-07 02:47:10.179396.
INFO:custodian.custodian:Run completed. Total time taken = 3:45:39.593648.
