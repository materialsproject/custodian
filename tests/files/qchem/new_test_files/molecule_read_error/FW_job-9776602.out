foobar
2018-07-06 23:01:27,698 INFO Hostname/IP lookup (this will take a few seconds)
2018-07-06 23:01:27,700 INFO Launching Rocket
2018-07-06 23:01:30,553 INFO RUNNING fw_id: 2072 in directory: /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-04-03-30-43-540608/launcher_2018-07-07-05-35-08-084164
2018-07-06 23:01:30,576 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-07-06 23:01:30,582 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-07-06 23:01:30,585 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
